src/TuxMutableRenderMixin.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+var invariant = require('./TuxInvariant');
+
+// findPath FUNCTION: recursive function to discover the first path to the provided @param mutableTrait from deepSearch function
+// @param mutableTrait STRING trait to find the path of keys to within the searchObject
+// @param searchObject OBJECT:
+  // expected keys ANY: this object can contain any keys, but is expected to contain the key specified in mutableTrait
+// @param currentPath ARRAY: array of strings that lead to the specified mutableTrait within searchObject
+// @param mutablePath ARRAY: empty array that the valid path will be added to once found
+var findPath = function (mutableTrait, searchObject, currentPath, mutablePath) {
+  // if the searchObject is undefined or null, return as the path isn't down this branch
+  if (searchObject === undefined || searchObject === null) {
+    return;
+  } else if (searchObject.hasOwnProperty(mutableTrait)) {
+    // Add mutableTrait to the end of the currentPath and set it to mutablePath and return
+    mutablePath = currentPath.push(mutableTrait);
+    return;
+  }
+
+  // declare reused variables
+  var key, newPath;
+
+  for (key in searchObject) {
+    // add the new key onto currentPath
+    newPath = currentPath.concat(key);
+    // recurse over the item at searchObject[key] with newPath as the currentPath input
+    findPath(searchObject[key], newPath);
```

Comment:
You need to pass in the `mutablePath` to each invocation of `findPath` and set `mutablePath` equal to the result of calling `findPath` here on line 28.